.changes/next-release/bugfix-04244d8a-3206-4b6f-97d8-15f82bdeeb53.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix a cloned run config's changing altering the original as well"
```

Comment:
Fix issue where modifying a cloned run config results in mutation of the original